hack/verify-payload-crds.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+source "$(dirname "${BASH_SOURCE}")/update-payload-crds.sh"
+
+files=""
+
+# Check there's no diff between the files in their canonical location
+# and the payload-manifests location.
+for glob in ${crd_globs}; do
+    for f in ${SCRIPT_ROOT}/${glob}; do
+        files+="$(basename ${f}),"
+        echo "Verifiying diff on $(basename $f)"
+        diff $f ${SCRIPT_ROOT}/payload-manifests/crds/$(basename $f)
```

Comment:
Same suggestions as for `hack/update-payload-crds.sh` plus a typo fix ("Verifiying"):
```suggestion
for f in ${crd_globs}; do
      files+="$(basename "${f}"),"
      echo "Verifying diff on $(basename "$f")"
      diff "$f" "${SCRIPT_ROOT}/payload-manifests/crds/$(basename "$f")"
```